Sources/PackageModel/Destination.swift

```diff
@@ -103,7 +103,7 @@ public struct Destination: Encodable, Equatable {
     }
     
     /// Additional flags to be passed to the build tools.
-    public let extraFlags: BuildFlags
+    public var extraFlags: BuildFlags
```

Comment:
does it need to be mutated?

Reply:
Yes, it happens on https://github.com/apple/swift-package-manager/pull/5922/files#diff-ba866b666cdc2795c59e6713e0817015f0076f5447541e8c477db717cbb6baf3R114

Since it's a struct, I think making this `var` is safe, as any mutations are local and don't have side effects?

Reply:
👍